packages/commonwealth/client/scripts/views/pages/notification_settings/index.tsx

```diff
@@ -56,6 +60,22 @@ const NotificationSettingsPage = () => {
     relevantSubscribedCommunities,
   } = useNotificationSettings();
 
+  const { mutateAsync: registerClientRegistrationToken } =
```

Comment:
Marcin wants all API queries to follow the standard pattern and be in the state/api folder (See createGroup.ts for example). We can then replace this direct call this wrapper

Reply:
I'm talking to him about it in another PR but I think this might no longer be necessary or desirable.  

One issue, for example, is that createGroup.ts doesn't useCallback so we're causing excessive React re-renders now because of this.  That can be fixed but these functions will end up being 1 liners.

Reply:
TBH I don't have a strong preference either way since I am not too front end focused. Can @mzparacha comment here if this should be changed? If we think we can just stick to the functions, then I think we should keep it going forward.

Reply:
I would suggest following the `state/api` pattern for consistency. I understand that this might be a few lines with trpc, but we do a lot of other things on success/error cases of a specific query like reset another query's data, or do some frontend state changes etc, this helps to repeat the same action whenever a specific query is run.

Reply:
as asked in another PR and as discussed already, please keep it in `state/api`, mainly for being well organised, having good readability and consistency